cmd/clusterctl/client/alpha/rollout_pauser_test.go

```diff
@@ -153,7 +153,7 @@ func Test_ObjectPauser(t *testing.T) {
 			}
 			g.Expect(err).ToNot(HaveOccurred())
 			for _, obj := range tt.fields.objs {
-				cl, err := proxy.NewClient()
+				cl, err := proxy.NewClient(context.TODO())
```

Comment:
I could see where `context.Background()` might be appropriate in tests if we don't plan to ever have some higher-level context that this test could inherit.